src/main/java/org/futurecollars/lesson10task1phoneBook/PhoneBook.java

```diff
@@ -0,0 +1,54 @@
+package org.futurecollars.lesson10task1phoneBook;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class PhoneBook {
+    private String fullName;
+    private String phoneNumber;
+    private String address;
+    public static List<PhoneBook> phoneBook = new ArrayList<>();
```

Comment:
Dlaczego klasa PhoneBook zawiera listę obiektów PhoneBook?
To tak jakby prawdziwa książka telefoniczna zawierała listę książek telefonicznych - nie brzmi to dobrze.